service/docs/source/security.rst

```diff
@@ -0,0 +1,525 @@
+
+Guide to Service Security
+=========================
+
+This documentation describes the objectives, mechanisms, assets and
+threats that were considered when designing the GEOPM Service security
+architecture.  This information may be useful for system
+administrators, developers, and users who would like to better
+understand how the GEOPM Service provides security guarantees.
+
+
+High Level Security Objectives
+------------------------------
+
+The GEOPM Service provides a secure gateway to privileged hardware
+features. This section describes the security expectations of the
+system administrator who deploys and configures the GEOPM Service as
+described in the GEOPM documentation.
+
+
+Fine-grained access management for hardware features
```

Comment:
You're mixing upper/lower case for these section titles.  Do you want it like this instead:
```suggestion
Fine-Grained Access Management for Hardware Features
```